packages/blade/src/tokens/theme/theme.d.ts

```diff
@@ -50,6 +50,10 @@ export type ActionStates = {
   disabled: string;
 };
 
+export type LinkActionStates = ActionStates & {
```

Comment:
@kamleshchandnani can you take a look at this once? Have updated the types to accommodate a new `visited` token